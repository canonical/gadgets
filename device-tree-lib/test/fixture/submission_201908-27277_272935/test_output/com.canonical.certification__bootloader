name: uboot
booted_kernel_path: unknown
booted_kernel_partition_type: unknown

