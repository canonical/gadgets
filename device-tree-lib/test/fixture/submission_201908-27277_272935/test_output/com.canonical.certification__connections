slot: snapd:bluetooth-control
plug: checkbox:bluetooth-control

slot: checkbox20:checkbox-runtime
plug: checkbox:checkbox-runtime

slot: snapd:network-bind
plug: checkbox:network-bind

slot: checkbox20:provider-certification-client
plug: checkbox:provider-certification-client

slot: checkbox20:provider-certification-server
plug: checkbox:provider-certification-server

slot: checkbox20:provider-checkbox
plug: checkbox:provider-checkbox

slot: checkbox20:provider-docker
plug: checkbox:provider-docker

slot: checkbox20:provider-gpgpu
plug: checkbox:provider-gpgpu

slot: checkbox20:provider-ipdt
plug: checkbox:provider-ipdt

slot: checkbox20:provider-resource
plug: checkbox:provider-resource

slot: checkbox20:provider-sru
plug: checkbox:provider-sru

slot: checkbox20:provider-tpm2
plug: checkbox:provider-tpm2

