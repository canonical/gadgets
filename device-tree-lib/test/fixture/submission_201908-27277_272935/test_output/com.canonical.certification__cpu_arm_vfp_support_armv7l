half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32
