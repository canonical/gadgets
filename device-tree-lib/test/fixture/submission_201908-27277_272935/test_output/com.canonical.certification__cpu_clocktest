Testing for clock jitter on 4 cpus
PASSED: largest jitter seen was 0.008161

Testing clock direction for 5 minutes...
PASSED: Iteration 0 delta: 0.000099
PASSED: Iteration 1 delta: 0.000103
PASSED: Iteration 2 delta: 0.000100
PASSED: Iteration 3 delta: 0.000110
PASSED: Iteration 4 delta: 0.000099
clock direction test: sleeptime 60 sec per iteration, failed iterations: 0
