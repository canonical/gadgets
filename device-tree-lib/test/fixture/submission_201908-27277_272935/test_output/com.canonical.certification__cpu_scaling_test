Skipped Tests: 1
 WARNING: The following test cases were skipped by fwts:
  - cpufreq

 Please review the following log for more information:

Results generated by fwts: Version V22.05.00 (2022-05-23 05:53:49).

Some of this work - Copyright (c) 1999 - 2021, Intel Corp. All rights reserved.
Some of this work - Copyright (c) 2010 - 2021, Canonical.
Some of this work - Copyright (c) 2016 - 2021, IBM.
Some of this work - Copyright (c) 2017 - 2021, ARM Ltd.

This test run on 16/07/22 at 12:29:37 on host Linux ubuntu 5.4.0-1066-raspi
#76-Ubuntu SMP PREEMPT Mon Jun 27 11:04:02 UTC 2022 armv7l.

Command: "fwts -q --stdout-summary -r /var/tmp/checkbox-ng/sessions
/session_title-2022-07-16T12.25.57.session/session-share/scaling_test.log
cpufreq".
Running tests: cpufreq.

cpufreq: CPU frequency scaling tests.
--------------------------------------------------------------------------------
Test 1 of 7: CPU frequency table consistency
PASSED: Test 1, CPU frequency tables are consistent

Test 2 of 7: CPU frequency table duplicates
PASSED: Test 2, No duplicates in CPU frequency table

Test 3 of 7: CPU frequency firmware limits
PASSED: Test 3, No BIOS limits imposed

Test 4 of 7: CPU frequency claimed maximum
PASSED: Test 4, CPU max frequencies OK

Test 5 of 7: CPU frequency SW_ANY control
SKIPPED: Test 5, Platform doesn't perform SW_ cpu frequency control

Test 6 of 7: CPU frequency SW_ALL control
SKIPPED: Test 6, Platform doesn't perform SW_ cpu frequency control

Test 7 of 7: CPU frequency performance tests.
PASSED: Test 7, CPU performance scaling OK

================================================================================
5 passed, 0 failed, 0 warning, 0 aborted, 2 skipped, 0 info only.
================================================================================


5 passed, 0 failed, 0 warning, 0 aborted, 2 skipped, 0 info only.

Test Failure Summary
================================================================================

Critical failures: NONE

High failures: NONE

Medium failures: NONE

Low failures: NONE

Other failures: NONE

Test           |Pass |Fail |Abort|Warn |Skip |Info |
---------------+-----+-----+-----+-----+-----+-----+
cpufreq        |    5|     |     |     |    2|     |
---------------+-----+-----+-----+-----+-----+-----+
Total:         |    5|    0|    0|    0|    2|    0|
---------------+-----+-----+-----+-----+-----+-----+

