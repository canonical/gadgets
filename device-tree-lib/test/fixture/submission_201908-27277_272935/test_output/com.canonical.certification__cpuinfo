bogomips: 77
cache: -1
count: 4
model: armv7l
model_number: 0x0
model_revision: 4
model_version: 7
other: half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32
platform: armv7l
speed: 1200
type: BCM2835
governors: conservative ondemand userspace powersave performance schedutil
scaling: supported
