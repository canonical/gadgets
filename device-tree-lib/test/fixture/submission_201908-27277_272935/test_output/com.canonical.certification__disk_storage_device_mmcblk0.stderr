Using uid:0, gid:0.
Writing a byte at a time...done
Writing intelligently...done
Rewriting...done
Reading a byte at a time...done
Reading intelligently...done
start 'em...done...done...done...done...done...
Create files in sequential order...done.
Stat files in sequential order...done.
Delete files in sequential order...done.
Create files in random order...done.
Stat files in random order...done.
Delete files in random order...done.
